# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/hotplug/hotplug-noinit.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN hotplug-2003_05_01-mine/Makefile hotplug-2003_05_01/Makefile
--- hotplug-2003_05_01-mine/Makefile	2003-07-10 21:52:39.000000000 -0400
+++ hotplug-2003_05_01/Makefile	2003-07-10 21:51:08.000000000 -0400
@@ -20,7 +20,6 @@
 TOPDIR=/
 DIRS= 	/sbin \
 	/etc/hotplug \
-	/etc/init.d/ \
 	/var/run/usb \
 	/usr/share/man/man8
 
@@ -52,7 +51,6 @@
 
 $(RELEASE_DIR).tar.gz: clean
 	@chmod +x sbin/hotplug
-	@chmod +x etc/rc.d/init.d/hotplug
 	@chmod +x etc/hotplug/*.agent etc/hotplug/*.rc
 	@-rm -rf $(RELEASE_DIR)
 	@mkdir $(RELEASE_DIR)
@@ -91,7 +89,6 @@
 	for F in etc/hotplug/{blacklist,usb.{user,hand,dist}map} ; do \
 	    ${INSTALL_DATA} $$F $(etcdir)/hotplug ; \
 	    done
-	${INSTALL_SCRIPT} -D etc/rc.d/init.d/hotplug $(etcdir)/init.d/hotplug
 	$(INSTALL_DATA) -D hotplug.8  $(mandir)/man8/hotplug.8
 	$(INSTALL) -d $(prefix)/var/log/hotplug $(prefix)/var/run
 	$(INSTALL) -d -m 700 $(prefix)/var/run/usb
